### ISSUE TITLE
[UX] Redesign the Themes page

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1103]
The enabled themes are listed vertically, in a single column while the disabled themes are listed in horizontal grid system that wraps rows ~~every 4 themes~~ in a responsive way as width permits. The enabled themes have their info and action links to their right, while the disabled ones below them.

I think we should convert this page to a responsive grid that lists all themes horizontally the same way that the disabled themes are listed (wrapping as needed, depending on screen width). That would take advantage of the horizontal space left black to the right of the (currently single) column of enabled themes and make the page shorter, so less scrolling. UX++

Sort of OT: If it were otherwise, I'd suggest to place themes in a table that has separate columns for theme name, description and actions, but the theme screenshot would make the table rows too tall and waste vertical space (more scrolling). This would allow the enable/disable links to be converted to a checkbox so that multiple themes can be enabled/disabled with a single page load.

~~Also OT: Do you guys feel that UX-wise the fieldset of setting the admin theme deserves to be at the top of the page there? If so, let me know and I'll file a separate issue for that.~~

**Comment by klonos** [comment]
Created a separate issue for moving the admin theme selection to the top: #1104

**Comment by jenlampton** [comment]
I agree the themes page needs a redesign, but I'd like to see some mockups before we start changing the code.

**Comment by jenlampton** [comment]
Marked https://github.com/backdrop/backdrop-issues/issues/1104 as a duplicate of this one, as we should tackle those problems as part of a redesign of the whole page.
